neo4j/io/__init__.py

```diff
@@ -788,21 +788,21 @@ def _secure(s, host, ssl_context):
     local_port = s.getsockname()[1]
     # Secure the connection if an SSL context has been provided
     if ssl_context:
-        log.debug("[#%04X]  C: <SECURE> %s", local_port, host)
+        log.debug("[#%04X]  C: <ENCRYPTED> %s", local_port, host)
```

Comment:
This log message doesn't need to change. SECURE is right here because we are applying the SSL/TLS layer.